Airwallex/Core/Sources/AWXDefaultProvider.h

```diff
@@ -114,6 +114,15 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)handleFlow;
 
+/**
+ Create a new payment consent and confirm the payment intent with payment method.
+
+ @param paymentMethod The payment method info.
+ @param device The current device info.
+ */
+- (void)createPaymentConsentAndConfirmIntentWithPaymentMethod:(AWXPaymentMethod *)paymentMethod
```

Comment:
Actually, what if you just use confirmPaymentIntentWithPaymentMethod:..paymentConsent:...device:... The method already takes care of creating the consent anyway. It does feel a bit weird that you can pass in a nullable payment consent though

Reply:
That method will directly confirm payment intent when session is `OneOffSession` (which is still needed for not saving card scenario), the aim of this method is to separate from that so we can create consent and then confirm. And save card logic shouldn't be in this class, so that's why I create another method.